documentation/asciidoc/topics/proc_installing_command_line.adoc

```diff
@@ -0,0 +1,66 @@
+[id='installing-openshift-command-line_{context}']
+= Installing the Helm chart on the command line
+
+[role="_abstract"]
+Use the `helm` and `oc` clients to install the {brandname} Helm chart from the Red Hat Helm Charts repository via the command line.
+ifdef::community[]
+This approach lets you automate the deployment of {brandname} clusters on {openshiftshort} or to install the {brandname} Helm chart on Kubernetes clusters.
+endif::community[]
+ifdef::downstream[]
+This approach lets you script the deployment of {brandname} clusters on {openshiftshort}.
+endif::downstream[]
+
+.Prerequisites
+
+ifdef::community[]
+* Have access to an {openshiftshort} or Kubernetes cluster.
+* Have an `oc` or `kubectl` client.
+endif::community[]
+ifdef::downstream[]
+* Have access to an {openshiftshort} cluster.
+* Have an `oc` client.
+endif::downstream[]
+* Have a `helm` client.
+* Add the link:https://github.com/redhat-developer/redhat-helm-charts/blob/master/README.md[Red Hat Helm Charts repository].
+
+.Procedure
+
+. Create a values file for your {brandname} cluster that you can pass to the Helm chart.
++
+For example, the following values file creates a cluster with two nodes:
++
+[source,options="nowrap",subs=attributes+]
+----
+$ cat > infinispan-values.yaml<<EOF
+include::yaml/infinispan-values.yaml[]
+EOF
+----
++
+[TIP]
+====
+To find descriptions for each of the {brandname} values, refer to the link:{helm_chart_readme}[{brandname} chart README].
+====
++
+. Log in to your {openshiftshort} cluster from the `oc` client.
+. Install the {brandname} chart and specify the values file for your cluster.
++
+[source,options="nowrap",subs=attributes+]
+----
+$ helm install infinispan redhat-charts/infinispan -f infinispan-values.yaml
```

Comment:
It might be necessary to include the `-n <namespace>` arg. As the `helm` CLI is independent of the `oc` client, I don't think it has a notion of current project, similar to the limitations of `kubectl`.